### ISSUE TITLE
Top bar and and "add" icon move when scrolling

### ISSUE THREAD

**Issue by baihuqian** [chenasraf/gInbox#37]
I installed gInbox today and notice a problem: the top search bar and red "add" icon moves up and down rapidly when scrolling, as though they are scrolled with the page and then are forced back to place. I don't know any Objective-C but can anyone fix this? It's very annoying. Thanks.

I'm running latest 10.10 on 2015 13'' RMBP with latest Chrome installed.

**Comment by chenasraf** [comment]
It's a problem with webkit... I've tried solving it but no real idea how. When I do I'll let you know :)

**Comment by jameswilson** [comment]
Yeah, this is strange because the inbox.google.com site works nicely in Chrome (which is webkit based).  Google must have added their own special sauce to prevent this.

**Comment by sergeych** [comment]
Did you try mimicking user agent?
